tests/CMakeLists.txt

```diff
@@ -178,7 +178,7 @@ if(BUILD_GENIVI)
                      src/logger.cc
                      tests/rvisotaclient_test.cc)
     add_executable(aktualizr_test_rvi_client ${TEST_RVI_SRC})
-    target_link_libraries(aktualizr_test_rvi_client rvi ${TEST_LIBS} gtest)
+    target_link_libraries(aktualizr_test_rvi_client ${TEST_LIBS} gtest)
```

Comment:
I think this is required: The CI build fails with "/source/src/sotarviclient.h:3:17: fatal error: rvi.h: No such file or directory"